en/Useful-Websites-to-Learn-How-to-Code-Quickly.md

```diff
@@ -0,0 +1,95 @@
+
+Language: JavaScript 
+4 lessons plus best practices/resources to teach you how to program using Javascript. 
+
+Each lesson has a video and detailed articles to make sure you understand them.  
+
+![LifeHacker](images/9-useful-LifeHacker.jpg)
+
+##  [CodeCademy](http://www.codecademy.com/)
+
+Language: JavaScript 
+
+This is an interactive form of instruction that gives you a bit of a grip on JavaScript. 
+
+For each course you complete, you get some achievement points and badges that will inspire your interest in learning. 
+
+![](images/9-useful-CodeCademy.jpg)
+
+
+## [TeamTreeHouse](http://teamtreehouse.com/)
+
+
+Languages: CSS, CSS3, HTML, HTML5, JavaScript, Basic Programming, and iOS Development 
+
+This is a paid member online learning service that covers topics such as Web design and development, as well as iOS development, which is now very popular. 
+
+![](images/9-useful-TeamTreeHouse.jpg)
+
+## [RubyMonk](http://rubymonk.com/)
+
+Language: Ruby RubyMonk is an interactive learning platform. 
+
+You can learn Ruby through courses, problem solving, or related articles. 
+
+![](images/9-useful-rubyMonk.jpg)
+
+## [Hackety](https://github.com/hacketyhack/hacketyhack)
+
+Language: Ruby 
+
+Hackety will teach you the absolute basics of programming, even if you don't have any programming experience. 
+
+![](images/9-useful-hackety.jpg)
+
+## [jQuery Air](http://jqueryair.com/)
+
+
+Language: JavaScript jQuery is the most famous JavaScript framework. Now you can learn jQuery directly in your browser. 
+
+The process of learning jQuery should be fun, and jQuery Air does it through a number of practical ways. 
+
+![](images/9-useful-jqueryAir.jpg)
+
+
+## [CodingBat](http://codingbat.com/)
+
+Language: Java, Python 
+
+Learn about Java and Python by solving sample problems and online coding practices. 
+
+![](images/9-useful-codingBat.jpg)
+
+## [PHP Know How](http://www.phpknowhow.com/)
+
+Language: PHP 
+
+This is not an interactive way to learn, but it has a good written tutorial that contains a wealth of examples and guides to teach you the basics of PHP and MySQL. 
+
+![](images/9-useful-phpKnowHow.jpg)
+
+## [MongoDB](http://www.mongodb.org/)
+
+Languages: Java, Python, Go, PHP and other mainstream development languages This is the official website of MongoDB. MongoDB is a scalable, high-performance, open-source NoSQL database written in the C? language. For java, Python, Go, PHP and other mainstream development languages. 
+
+The site contains a tutorial that lets you start your MongoDB learning journey with some commands. 
+
+![](images/9-useful-mongodb.jpg)
+
+## [学堂在线](https://next.xuetangx.com/)
+
+Technology: WeChat small program development, algorithm This is an online education website supported by Tsinghua University and other institutions.
+
+It includes courses such as recommended algorithms, WeChat mini-program production, etc. 
```

Comment:
It includes courses such as recommendation algorithms, WeChat mini-program development, etc.